test/ui-automation/wdio.conf.js

```diff
@@ -13,8 +13,7 @@ const mockDemoDomain = "https://demo-adapter.trustbloc.local:8094"
 
 exports.config = {
   ...config,
-  // TODO: changed this to url - fix wallet name issue
-  walletName: "wallet.trustbloc.local",
+  walletName: "TrustBloc Wallet - Local",
```

Comment:
Just curious - does your change fix this issue or it just started working on its own in CHAPI?

Reply:
I didn't add anything specific in this PR for this. In local wallet, I can see the name of Wallet (even automation passes).